src/ng/q.js

```
@@ -361,6 +364,8 @@ function qFactory(nextTick, exceptionHandler) {
         } else {
           this.promise.$$state.value = val;
           this.promise.$$state.status = 1;
+
+          promiseTracker && promiseTracker.untrack(this.promise);
```

Really not necessary as this tracker is declared as a required dep. If you dont get it you are only gonna fail silently which is probably not a good thing. Same here and everywhere. I get the defensive coding but in all reality..

---

removed

---

removed